Changelog.rst

```diff
@@ -8,34 +8,35 @@ Changelog
 **Bug Fixes**
 
   * The ``restore_check`` function did not work properly with wildcard index
-    patterns.  This has been rectified, and an integration test added to 
+    patterns.  This has been rectified, and an integration test added to
     satisfy this.  Reported in #989 (untergeek)
+  * Re-use cached response for nodes in most_available_node function
```

Comment:
Can you mention the PR # and your github handle in parenthesis, like the other lines?

Reply:
👍